pkg/runner/local_docker.go

```diff
@@ -400,6 +435,10 @@ func ensureSidecarContainer(ctx context.Context, cli *client.Client, log *zap.Su
 				Type:   mount.TypeBind,
 				Source: "/var/run/docker.sock", // TODO: don't hardcode this.
 				Target: "/var/run/docker.sock",
+			}, {
```

Comment:
I missed that during review - we should be consistent with the way we start the `sidecar` and update it in every runner when changing - currently we have Docker Swarm playbooks, and Kubernetes yaml specs in addition to this. Any other places we start the sidecar?

cc @raulk @Stebalien